resolvers/user.js

```diff
@@ -0,0 +1,10 @@
+import * as dbUsers from "../mongo/user";
```

Comment:
I'd suggest we add dataloaders here instead. They add a significant performance enhancement and add a level of abstraction. See https://github.com/facebook/dataloader and https://github.com/freeCodeCamp/open-api/issues/38